mmf/configs/zoo/datasets.yaml

```diff
@@ -543,6 +543,36 @@ localized_narratives:
         - url: https://storage.googleapis.com/localized-narratives/annotations/open_images_test_localized_narratives.jsonl
           file_name: open_images_test_localized_narratives.jsonl
           compressed: false
+        - url: https://storage.googleapis.com/localized-narratives/annotations/open_images_train_v6_captions.jsonl
```

Comment:
Should we split this into a separate zoo node like `localized_narratives.only_captions` or something? What do you think?

Reply:
sounds good. Will update.